t/app/controller/admin/users.t

```diff
@@ -550,6 +550,26 @@ FixMyStreet::override_config {
         $mech->content_contains( 'Updated!' );
     };
 
+    subtest "Test edit user name who has verified landline" => sub {
```

Comment:
might be good to add in a "with phone number unchanged" to the description here to make it a bit clearer what's being tested

Reply:
Done